src/StackAdmin/Azs.Backup.Admin/Module/Azs.Backup.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Set-AzsBackupShare.ps1

```diff
@@ -68,34 +77,53 @@ function Set-AzsBackupShare {
         [System.String]
         $Location,
 
-        [Parameter(Mandatory = $true, ParameterSetName = 'ResourceId')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'ResourceId')]
         [Parameter(Mandatory = $false, ParameterSetName = 'InputObject')]
-        [Parameter(Mandatory = $true, ParameterSetName = 'Update')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'Update')]
         [ValidateNotNullOrEmpty()]
         [System.String]
+        [Alias("Path")]
         $BackupShare,
 
-        [Parameter(Mandatory = $true, ParameterSetName = 'ResourceId')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'ResourceId')]
         [Parameter(Mandatory = $false, ParameterSetName = 'InputObject')]
-        [Parameter(Mandatory = $true, ParameterSetName = 'Update')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'Update')]
         [ValidateNotNullOrEmpty()]
         [System.String]
         $Username,
 
-        [Parameter(Mandatory = $true, ParameterSetName = 'ResourceId')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'ResourceId')]
         [Parameter(Mandatory = $false, ParameterSetName = 'InputObject')]
-        [Parameter(Mandatory = $true, ParameterSetName = 'Update')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'Update')]
         [ValidateNotNull()]
         [securestring]
         $Password,
 
-        [Parameter(Mandatory = $true, ParameterSetName = 'ResourceId')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'ResourceId')]
         [Parameter(Mandatory = $false, ParameterSetName = 'InputObject')]
-        [Parameter(Mandatory = $true, ParameterSetName = 'Update')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'Update')]
         [ValidateNotNull()]
         [securestring]
         $EncryptionKey,
 
+        [Parameter(Mandatory = $false, ParameterSetName = 'ResourceId')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'InputObject')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'Update')]
+        [bool]
+        $IsBackupSchedulerEnabled,
+
+        [Parameter(Mandatory = $false, ParameterSetName = 'ResourceId')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'InputObject')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'Update')]
+        [int]
+        $BackupFrequencyInHours,
```

Comment:
@deathly809 should the `-BackupFrequencyInHours` and `-BackupRetentionPeriodInDays` parameters only be set if the user provides `-EnableBackupScheduler`? Or could the user set these values if the `IsBackupSchedulerEnabled` property is set to true?

Reply:
@cormacpayne Users can set -BackupFrequencyInHours and -BackupRetentionPeriodInDays even if IsBackupSchedulerEnabled is false. According to the feature design, when IsBackupSchedulerEnabled set to false, scheduled backup and deletion will be skipped, but users can still do configuration.